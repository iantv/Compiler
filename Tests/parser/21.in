void a(int b){
	1 + b;
}