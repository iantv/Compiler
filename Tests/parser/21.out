main: function
	b: parameter
		int
	{
		expression:
				b
			+
				1
	}
returns
	void
